libs/ledger-live-common/src/api/Ledger.ts

```diff
@@ -12,31 +11,18 @@ type LedgerExplorer = {
 export const findCurrencyExplorer = (
   currency: CryptoCurrency
 ): LedgerExplorer | null | undefined => {
-  const config = getExplorerConfig()[currency.id];
-  if (!config) return;
-  const { id } = config;
   if (getEnv("SATSTACK") && currency.id === "bitcoin") {
     return {
       endpoint: getEnv("EXPLORER_SATSTACK"),
       id: "btc",
       version: "v3",
     };
   }
-  if (config.experimental && getEnv("EXPERIMENTAL_EXPLORERS")) {
-    const base = config.experimental.base;
-    const version = config.experimental.version;
-    return {
-      endpoint: getEnv(base),
-      id,
-      version,
-    };
-  }
 
-  const { base, version } = config.stable;
   return {
-    endpoint: getEnv(base),
-    id,
-    version,
+    endpoint: getEnv("EXPLORER"),
+    id: currency.ticker.toLocaleLowerCase(),
```

Comment:
dangerous assumption. It would be better to either maintain a separate list OR have backend/infra supporting our actual "currency id", because ticker could change over time too and are more related to market/display on the ui.

also toLocaleLowerCase() have different behavior depending on clients, would prefer toLowerCase()

Reply:
my problem was that the overriding config had btc, eth, ltc, ... as ids and the actual ids are bitcoin, ethereum, litecoin, ... I can maybe add a parameter to the currency ?

Reply:
Done